src/transform/index.ts

```diff
@@ -120,15 +121,17 @@ export const transform = () => {
         }
       }
 
-      code = writeBlock(Array.from(fileReferences, (ref) => `/// <reference path="${ref}" />`));
+      let code = writeBlock(Array.from(fileReferences, (ref) => `/// <reference path="${ref}" />`));
       code += writeBlock(Array.from(typeReferences, (ref) => `/// <reference types="${ref}" />`));
       code += fixer.fix();
 
       if (!code) {
         code += "\nexport { }";
       }
 
-      return { code, map: { mappings: "" } };
+      const exportsFixer = new ExportsFixer(parse(chunk.fileName, code));
```

Comment:
Now the `renderChunk` hook will parse the generated code twice. I believe its fine for now, but might be worth to merge these two post-processors into a single pass.

Reply:
Yes, I thought about doing it in one pass, but ultimately decided that it is cleaner to separate the postprocessing.